R/robustbayesianmetaanalysis.R

```diff
@@ -616,22 +613,29 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     if (!is.null(dataset)) {
       return(dataset)
     } else{
-      var_names <-
+      varNames <-
         c(
-          options[["input_t"]],
-          options[["input_ES"]],
-          options[["input_SE"]],
-          options[["input_N"]],
-          options[["input_N1"]],
-          options[["input_N2"]],
-          unlist(options[["input_CI"]])
+          options[["inputT"]],
+          options[["inputES"]],
+          options[["inputSE"]],
+          options[["inputN"]],
+          options[["inputN1"]],
+          options[["inputN2"]],
+          unlist(options[["inputCI"]])
         )
-      var_names <- var_names[var_names != ""]
+      varNames <- varNames[varNames != ""]
       
       dataset <- readDataSetToEnd(
-        columns.as.numeric = var_names,
-        columns = if (options[["input_labels"]] != "") options[["input_labels"]]
+        columns.as.numeric = varNames,
+        columns = if (options[["inputLabels"]] != "") options[["inputLabels"]]
       )
+
+      if (options[["inputLabels"]] != ""){
+        dataset[[.v(options[["inputLabels"]])]] <- as.character(dataset[[.v(options[["inputLabels"]])]])
```

Comment:
Weren't we getting rid of all these .v and .unv functions? Is there a reason why it's needed here?

Reply:
Perhaps consults Joris about this.

Reply:
I will ask him

Reply:
So what's the outcome of the .v and .unv conversation?

Reply:
I talked to Tim and they are turned off now and not used anymore. So I just deleted them.